v2.1/add-column.md

```diff
@@ -74,7 +74,7 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 ~~~
 
-### Add a Non-Null Column with a Default Value
+### Add a non-NULL column with a default value
```

Comment:
Should this be "Add a n non-`NULL` column..."? Or even "Add a `NOT NULL` column..."

Reply:
Done.